azext_iot/tests/__init__.py

```diff
@@ -81,6 +84,14 @@ def __init__(self, test_scenario, entity_name, entity_rg):
 
         super(IoTLiveScenarioTest, self).__init__(test_scenario)
 
+        if self.entity_name != settings.env.azext_iot_testhub:
+            self.cmd(
+                "iot hub create --name {} --resource-group {} --fc {} --fcs {} --sku S1 ".format(
```

Comment:
This is forcing every new iothub created to integrate with a storage account, even for tests that don't have any dependency on storage. Ideally the integration with storage should happen on the storage related tests/tests that have a dependency on iothub configuration with storage.